mismi-s3/test/Test/Mismi/S3.hs

```diff
@@ -66,3 +75,24 @@ withToken t f = do
   u <- liftIO $ T.pack . U.toString <$> U.nextRandom
   let a = Address b (Key . T.intercalate "/" $ ["mismi", u, unToken t])
   awsBracket_ (pure ()) (listRecursively a >>= mapM_ delete >> delete a) (f a)
+
+
+newAddress :: Token -> AWS Address
```

Comment:
Yeh. I think. I prefer the model we adopted in the scala code. Env vars to turn on debugging (i.e. log files and don't register delete finalizer), and just drop the slightly wierd coupling to quick-check.